src/components/common/layout/base.njk

```diff
@@ -105,6 +104,7 @@
 {% endblock %}
 
 {% block footer %}
+{% if webChannel %}
```

Comment:
Hi @SophiaDWokoma - I'm a little nervous about the footer presence being reliant upon `webChannel`. I say this because authentication frontend is a Web application, so the Web presentation should therefore be the default. 

What would your view be on the possibility of inverting the condition here, so the footer is included wherever the channel is not mobile? I'm thinking of something along these lines

```pseudocode
if mobileChannel === true
    do nothing
else
    include footer
```

I'd also suggest using strict equality because we've had problems previously where it's not been used (there's more detail in [this PR](https://github.com/govuk-one-login/authentication-frontend/pull/1109))